src/Controller/AdminController.php

```diff
@@ -30,10 +31,41 @@ public function __construct()
      */
     public function index(): string
     {
+        session_start();
```

Comment:
No put your session_start() way higher in the stackcall, or you'll have to repeat it everywhere. 
The best place is usually in public/index.php